app/services/description_validator.rb

```diff
@@ -2,23 +2,60 @@
 
 # Validate the descriptive metadata spreadsheet
 class DescriptionValidator
-  def initialize(csv)
+  def initialize(csv, bulk_job: false)
+    @csv = csv
     @headers = csv.headers
+    @bulk_job = bulk_job # indicates if validating from a bulk job
     @errors = []
   end
 
   attr_reader :errors
 
   def valid?
+    validate_duplicate_headers
+    validate_title_headers
+    if @bulk_job
+      validate_druid_headers
+      validate_druid_rows
+    end
+    errors.empty?
+  end
+
+  def validate_duplicate_headers
     duplicate_headers.each do |header|
       errors << "Duplicate column headers: The header #{header} should occur only once."
     end
-    errors.empty?
+  end
+
+  def validate_title_headers
+    if @headers.include?('title1.value') ||
+       (@headers.exclude?('title1.value') && @headers.include?('title1.structureValue1.type') && @headers.include?('title1.structureValue1.value'))
```

Comment:
Does this need to work for titleN (title1, title2, ...)?

Reply:
We are hoping that it's reasonable to expect a user would first supply a title1 before supplying a title2, and that if they didn't do this, getting a validation error would prompt them to do it.  This simplifying assumption makes the code easier to understand.